# Portuguese (Brazil) translation for apport
# Copyright (c) (c) 2006 Canonical Ltd, and Rosetta Contributors 2006
# This file is distributed under the same license as the apport package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: apport\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2013-11-19 08:43+0100\n"
"PO-Revision-Date: 2012-04-13 13:37+0000\n"
"Last-Translator: Neliton Pereira Jr. <nelitonpjr@gmail.com>\n"
"Language-Team: Portuguese (Brazil) <pt_BR@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2014-04-04 04:38+0000\n"
"X-Generator: Launchpad (build 16976)\n"
"Language: \n"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:124
#: ../apport/ui.py:124
msgid "This package does not seem to be installed correctly"
msgstr "Este pacote não parece estar instalado corretamente"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:128
#: ../apport/ui.py:128
#, python-format
msgid ""
"This is not an official %s package. Please remove any third party package "
"and try again."
msgstr ""
"Este não é um pacote oficial do %s. Por favor, remova qualquer pacote de "
"terceiros e tente novamente."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:145
#: ../apport/ui.py:145
#, python-format
msgid ""
"You have some obsolete package versions installed. Please upgrade the "
"following packages and check if the problem still occurs:\n"
"\n"
"%s"
msgstr ""
"Você tem alguns pacotes obsoletos instalados. Por favor atualize os pacotes "
"a seguir e verifique se o problema continua a ocorrer:\n"
"\n"
"%s"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:269
#: ../apport/ui.py:269
msgid "unknown program"
msgstr "programa desconhecido"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:270
#: ../apport/ui.py:270
#, python-format
msgid "Sorry, the program \"%s\" closed unexpectedly"
msgstr "Desculpe, o programa \"%s\" fechou inesperadamente"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:272
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1306
#: ../apport/ui.py:272 ../apport/ui.py:1306
#, python-format
msgid "Problem in %s"
msgstr "Problema em %s"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:273
#: ../apport/ui.py:273
msgid ""
"Your computer does not have enough free memory to automatically analyze the "
"problem and send a report to the developers."
msgstr ""
"Seu computador não possui memória suficiente para analisar automaticamente o "
"problema e enviar um relatório aos desenvolvedores."

#. package does not exist
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:288
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:294
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:300
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:343
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:351
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:478
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:481
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:682
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1113
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1278
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1282
#: ../debian/tmp/usr/share/apport/testsuite/test_ui.py:276
#: ../debian/tmp/usr/share/apport/testsuite/test_ui.py:1049
#: ../apport/ui.py:288 ../apport/ui.py:294 ../apport/ui.py:300
#: ../apport/ui.py:343 ../apport/ui.py:351 ../apport/ui.py:478
#: ../apport/ui.py:481 ../apport/ui.py:682 ../apport/ui.py:1113
#: ../apport/ui.py:1278 ../apport/ui.py:1282
msgid "Invalid problem report"
msgstr "Relatório do problema inválido"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:289
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1284
#: ../apport/ui.py:289 ../apport/ui.py:1284
msgid "This problem report is damaged and cannot be processed."
msgstr "Este relatório de erros está danificado e não pode ser processado."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:295
#: ../apport/ui.py:295
msgid "The report belongs to a package that is not installed."
msgstr "O relatório pertence a um pacote que não está instalado."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:301
#: ../apport/ui.py:301
msgid "An error occurred while attempting to process this problem report:"
msgstr "Um erro ocorreu ao tentar processar esse relatório de problema:"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:344
#: ../apport/ui.py:344
msgid "You are not allowed to access this problem report."
msgstr "Você não tem permissão para acessar esse relatório de problema."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:347
#: ../apport/ui.py:347
msgid "Error"
msgstr "Erro"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:348
#: ../apport/ui.py:348
msgid "There is not enough disk space available to process this report."
msgstr "Não há espaço suficiente em disco para processar este relatório."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:432
#: ../apport/ui.py:432
msgid "No package specified"
msgstr "Nenhum pacote especificado"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:433
#: ../apport/ui.py:433
msgid ""
"You need to specify a package or a PID. See --help for more information."
msgstr ""
"Você precisa especificar o pacote ou o PID. Veja --help para mais "
"informações."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:456
#: ../apport/ui.py:456
msgid "Permission denied"
msgstr "Permissão negada"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:457
#: ../apport/ui.py:457
msgid ""
"The specified process does not belong to you. Please run this program as the "
"process owner or as root."
msgstr ""
"O processo especificado não pertence a você. Por favor, execute este "
"programa como dono do processo ou como usuário root."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:459
#: ../apport/ui.py:459
msgid "Invalid PID"
msgstr "PID inválido"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:460
#: ../apport/ui.py:460
msgid "The specified process ID does not belong to a program."
msgstr "O ID do processo especificado não pertence a um programa."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:479
#: ../apport/ui.py:479
#, python-format
msgid "Symptom script %s did not determine an affected package"
msgstr "O script de sintoma %s não determinou um pacote afetado."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:482
#: ../apport/ui.py:482
#, python-format
msgid "Package %s does not exist"
msgstr "O pacote %s não existe."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:506
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:694
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:699
#: ../apport/ui.py:506 ../apport/ui.py:694 ../apport/ui.py:699
msgid "Cannot create report"
msgstr "Não foi possível criar o relatório"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:521
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:567
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:584
#: ../apport/ui.py:521 ../apport/ui.py:567 ../apport/ui.py:584
msgid "Updating problem report"
msgstr "Atualizando o relatório de problema"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:522
#: ../apport/ui.py:522
msgid ""
"You are not the reporter or subscriber of this problem report, or the report "
"is a duplicate or already closed.\n"
"\n"
"Please create a new report using \"apport-bug\"."
msgstr ""
"Você não é o relator ou assinante deste relatório de problema, o relatório "
"está duplicado ou já foi fechado.\n"
"Por favor, crie um novo relatório utilizando o \"apport-bug\""

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:531
#: ../apport/ui.py:531
msgid ""
"You are not the reporter of this problem report. It is much easier to mark a "
"bug as a duplicate of another than to move your comments and attachments to "
"a new bug.\n"
"\n"
"Subsequently, we recommend that you file a new bug report using \"apport-"
"bug\" and make a comment in this bug about the one you file.\n"
"\n"
"Do you really want to proceed?"
msgstr ""
"Você não é o relator do presente relatório de problema. É muito mais fácil "
"marcar um bug como uma duplicação de outro do que para mover os seus "
"comentários e anexos de um novo bug.\n"
"\n"
"Posteriormente, recomendamos que você relate um bug novo usando o \"apport-"
"bug\" e fazer um comentário a este erro sobre o que um arquivo.\n"
"\n"
"Você realmente deseja continuar?"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:568
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:585
#: ../apport/ui.py:568 ../apport/ui.py:585
msgid "No additional information collected."
msgstr "Nenhuma informação adicional foi coletada."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:636
#: ../apport/ui.py:636
msgid "What kind of problem do you want to report?"
msgstr "Que tipo de problema você deseja informar?"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:653
#: ../apport/ui.py:653
msgid "Unknown symptom"
msgstr "Sintoma desconhecido"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:654
#: ../apport/ui.py:654
#, python-format
msgid "The symptom \"%s\" is not known."
msgstr "O sintoma \"%s\" não é conhecido."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:685
#: ../apport/ui.py:685
msgid ""
"After closing this message please click on an application window to report a "
"problem about it."
msgstr ""
"Depois de fechar esta mensagem, por favor, clique na janela do aplicativo "
"para relatar um problema sobre ele."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:695
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:700
#: ../apport/ui.py:695 ../apport/ui.py:700
msgid "xprop failed to determine process ID of the window"
msgstr "xprop falhou ao determinar a ID do processo da janela"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:714
#: ../apport/ui.py:714
msgid "%prog <report number>"
msgstr "%prog <relato número>"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:716
#: ../apport/ui.py:716
msgid "Specify package name."
msgstr "Especificar o nome do pacote."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:718
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:769
#: ../apport/ui.py:718 ../apport/ui.py:769
msgid "Add an extra tag to the report. Can be specified multiple times."
msgstr ""
"Adicione um rótulo a mais ao relatório. Pode ser especificado várias vezes."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:748
#: ../apport/ui.py:748
msgid ""
"%prog [options] [symptom|pid|package|program path|.apport/.crash file]"
msgstr ""
"%prog [opções] [sintoma|pid|pacote|localização do programa|arquivo "
".apport/.crash]"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:751
#: ../apport/ui.py:751
msgid ""
"Start in bug filing mode. Requires --package and an optional --pid, or just "
"a --pid. If neither is given, display a list of known symptoms. (Implied if "
"a single argument is given.)"
msgstr ""
"Iniciar em modo de classificação de bug. Requer --package e um opcional --"
"pid, ou somente um --pid. Se nenhum é fornecido, mostra uma lista de "
"sintomas conhecidos. (Automático se um único argumento é fornecido)"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:753
#: ../apport/ui.py:753
msgid "Click a window as a target for filing a problem report."
msgstr "Clique na janela como alvo para preencher um relatório de problema."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:755
#: ../apport/ui.py:755
msgid "Start in bug updating mode. Can take an optional --package."
msgstr ""
"Iniciar em modo de atualização de erro. Aceita o parâmetro --package."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:757
#: ../apport/ui.py:757
msgid ""
"File a bug report about a symptom. (Implied if symptom name is given as only "
"argument.)"
msgstr ""
"Submete um relatório de bug sobre um sintoma. (Automático se um nome de "
"sintoma é fornecido como único argumento)"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:759
#: ../apport/ui.py:759
msgid ""
"Specify package name in --file-bug mode. This is optional if a --pid is "
"specified. (Implied if package name is given as only argument.)"
msgstr ""
"Especifica um nome de pacote em modo --file-bug. Opcional se um --pid é "
"especificado. (Automático se um nome de pacote é fornecido como único "
"argumento)"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:761
#: ../apport/ui.py:761
msgid ""
"Specify a running program in --file-bug mode. If this is specified, the bug "
"report will contain more information.  (Implied if pid is given as only "
"argument.)"
msgstr ""
"Especifique um programa em funcionamento no modo --file-bug. Desta forma, o "
"relatório de erro conterá mais informações. (Implícito se pid for dado como "
"único argumento.)"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:763
#: ../apport/ui.py:763
msgid "The provided pid is a hanging application."
msgstr "O pid fornecido é de uma aplicação pendente."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:765
#: ../apport/ui.py:765
#, python-format
msgid ""
"Report the crash from given .apport or .crash file instead of the pending "
"ones in %s. (Implied if file is given as only argument.)"
msgstr ""
"Relatar a falha de um arquivo .apport ou .crash fornecido ao invés dos "
"arquivos pendentes em %s. (Automático se o arquivo é fornecido como único "
"argumento)"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:767
#: ../apport/ui.py:767
msgid ""
"In bug filing mode, save the collected information into a file instead of "
"reporting it. This file can then be reported later on from a different "
"machine."
msgstr ""
"No modo de arquivamento de erro, salve as informações coletadas em um "
"arquivo em vez de relatá-las. Esse arquivo pode ser reportado mais tarde ou "
"de outra máquina."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:771
#: ../apport/ui.py:771
msgid "Print the Apport version number."
msgstr "Imprimir o número da versão do Apport."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:911
#: ../apport/ui.py:911
msgid ""
"This will launch apport-retrace in a terminal window to examine the crash."
msgstr ""
"Isto irá executar o apport-retrace em uma janela de Terminal para examinar a "
"falha."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:912
#: ../apport/ui.py:912
msgid "Run gdb session"
msgstr "Executar sessão gdb"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:913
#: ../apport/ui.py:913
msgid "Run gdb session without downloading debug symbols"
msgstr "Executar sessão gdb sem baixar símbolos de depuração"

#. TRANSLATORS: %s contains the crash report file name
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:915
#: ../apport/ui.py:915
#, python-format
msgid "Update %s with fully symbolic stack trace"
msgstr "Atualizar %s com rastreamento de pilha totalmente simbólico"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:991
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1001
#: ../apport/ui.py:991 ../apport/ui.py:1001
msgid ""
"This problem report applies to a program which is not installed any more."
msgstr ""
"Este problema relatado se aplica a um programa que não está mais instalado."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1016
#: ../apport/ui.py:1016
#, python-format
msgid ""
"The problem happened with the program %s which changed since the crash "
"occurred."
msgstr ""
"O problema aconteceu com o programa %s que mudou desde que o problema "
"ocorreu."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1114
#: ../apport/ui.py:1114
msgid "Could not determine the package or source package name."
msgstr "Não foi possível determinar o pacote ou nome do pacote fonte."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1132
#: ../apport/ui.py:1132
msgid "Unable to start web browser"
msgstr "Não é possível iniciar navegador web"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1133
#: ../apport/ui.py:1133
#, python-format
msgid "Unable to start web browser to open %s."
msgstr "Não é possível iniciar navegador web para abrir %s."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1232
#: ../apport/ui.py:1232
#, python-format
msgid "Please enter your account information for the %s bug tracking system"
msgstr ""
"Por favor digite as informações da sua conta para o sistema de monitoramento "
"de falhas %s"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1244
#: ../apport/ui.py:1244
msgid "Network problem"
msgstr "Problema na rede"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1246
#: ../apport/ui.py:1246
msgid ""
"Cannot connect to crash database, please check your Internet connection."
msgstr ""
"Não é possível conectar em uma base de dados quebrada, por favor verifique "
"sua conexão com a internet."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1273
#: ../apport/ui.py:1273
msgid "Memory exhaustion"
msgstr "Memória exaurida"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1274
#: ../apport/ui.py:1274
msgid "Your system does not have enough memory to process this crash report."
msgstr ""
"Seu sistema não tem memória o suficiente para processar este relatório de "
"erro."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1309
#: ../apport/ui.py:1309
#, python-format
msgid ""
"The problem cannot be reported:\n"
"\n"
"%s"
msgstr ""
"O problema não pode ser relatado:\n"
"\n"
"%s"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1365
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1372
#: ../apport/ui.py:1365 ../apport/ui.py:1372
msgid "Problem already known"
msgstr "Problema já informado"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1366
#: ../apport/ui.py:1366
msgid ""
"This problem was already reported in the bug report displayed in the web "
"browser. Please check if you can add any further information that might be "
"helpful for the developers."
msgstr ""
"Este problema já foi relatado no relatório do erro mostrado no navegador. "
"Por favor, verifique se você pode adicionar alguma informação adicional que "
"possa ser útil aos desenvolvedores."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1373
#: ../apport/ui.py:1373
msgid "This problem was already reported to developers. Thank you!"
msgstr "O problema já foi reportado aos desenvolvedores. Obrigado!"

#: ../debian/tmp/usr/bin/apport-cli.py:74 ../bin/apport-cli.py:74
msgid "Press any key to continue..."
msgstr "Pressione qualquer tecla para continuar..."

#: ../debian/tmp/usr/bin/apport-cli.py:81 ../bin/apport-cli.py:81
msgid "What would you like to do? Your options are:"
msgstr "O que você gostaria de fazer? Suas opções são:"

#: ../debian/tmp/usr/bin/apport-cli.py:85 ../bin/apport-cli.py:85
#, python-format
msgid "Please choose (%s):"
msgstr "Por favor escolha (%s):"

#: ../debian/tmp/usr/bin/apport-cli.py:148 ../bin/apport-cli.py:148
#, python-format
msgid "(%i bytes)"
msgstr "(%i bytes)"

#: ../debian/tmp/usr/share/apport/apport-gtk.py:142 ../kde/apport-kde.py:342
#: ../debian/tmp/usr/share/apport/apport-kde.py:342
#: ../debian/tmp/usr/bin/apport-cli.py:150 ../gtk/apport-gtk.py:142
#: ../bin/apport-cli.py:150
msgid "(binary data)"
msgstr "(dados binários)"

#: ../debian/tmp/usr/share/apport/apport-gtk.py:177 ../kde/apport-kde.py:162
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_kde.py:286
#: ../debian/tmp/usr/share/apport/apport-kde.py:162
#: ../debian/tmp/usr/bin/apport-cli.py:178 ../gtk/apport-gtk.py:177
#: ../bin/apport-cli.py:178
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:404
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:429
msgid "Send problem report to the developers?"
msgstr "Enviar o relatório do problema aos desenvolvedores?"

#: ../debian/tmp/usr/bin/apport-cli.py:179 ../bin/apport-cli.py:179
msgid ""
"After the problem report has been sent, please fill out the form in the\n"
"automatically opened web browser."
msgstr ""
"Depois que o relatório do problema for enviado, por favor preencha o \n"
"formulário que será aberto automaticamente no navegador web."

#: ../debian/tmp/usr/bin/apport-cli.py:182 ../bin/apport-cli.py:182
#, python-format
msgid "&Send report (%s)"
msgstr "&Enviar relatório (%s)"

#: ../debian/tmp/usr/bin/apport-cli.py:186 ../bin/apport-cli.py:186
msgid "&Examine locally"
msgstr "&Examinar localmente"

#: ../debian/tmp/usr/bin/apport-cli.py:190 ../bin/apport-cli.py:190
msgid "&View report"
msgstr "&Ver relatório"

#: ../debian/tmp/usr/bin/apport-cli.py:191 ../bin/apport-cli.py:191
msgid "&Keep report file for sending later or copying to somewhere else"
msgstr ""
"&Manter arquivo do relatório para enviar mais tarde ou copiar para outro "
"lugar."

#: ../debian/tmp/usr/bin/apport-cli.py:192 ../bin/apport-cli.py:192
msgid "Cancel and &ignore future crashes of this program version"
msgstr "Cancelar e &ignorar futuros travamentos dessa versão do programa."

#: ../debian/tmp/usr/bin/apport-cli.py:194
#: ../debian/tmp/usr/bin/apport-cli.py:271
#: ../debian/tmp/usr/bin/apport-cli.py:303
#: ../debian/tmp/usr/bin/apport-cli.py:324 ../bin/apport-cli.py:194
#: ../bin/apport-cli.py:271 ../bin/apport-cli.py:303 ../bin/apport-cli.py:324
msgid "&Cancel"
msgstr "&Cancelar"

#: ../debian/tmp/usr/bin/apport-cli.py:222 ../bin/apport-cli.py:222
msgid "Problem report file:"
msgstr "Arquivo do relatório de problema:"

#: ../debian/tmp/usr/bin/apport-cli.py:228
#: ../debian/tmp/usr/bin/apport-cli.py:233 ../bin/apport-cli.py:228
#: ../bin/apport-cli.py:233
msgid "&Confirm"
msgstr "&Confirmar"

#: ../debian/tmp/usr/bin/apport-cli.py:232 ../bin/apport-cli.py:232
#, python-format
msgid "Error: %s"
msgstr "Erro: %s"

#: ../kde/apport-kde.py:387 ../debian/tmp/usr/share/apport/apport-kde.py:387
#: ../debian/tmp/usr/bin/apport-cli.py:238 ../bin/apport-cli.py:238
msgid "Collecting problem information"
msgstr "Coletando informações sobre o problema"

#: ../debian/tmp/usr/bin/apport-cli.py:239 ../bin/apport-cli.py:239
msgid ""
"The collected information can be sent to the developers to improve the\n"
"application. This might take a few minutes."
msgstr ""
"A informação coletada pode ser enviada aos desenvolvedores para melhorar\n"
"o aplicativo. Isso pode demorar alguns minutos."

#: ../gtk/apport-gtk.ui.h:13 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:13
#: ../kde/apport-kde.py:415 ../debian/tmp/usr/share/apport/apport-kde.py:415
#: ../debian/tmp/usr/bin/apport-cli.py:251 ../bin/apport-cli.py:251
msgid "Uploading problem information"
msgstr "Enviando informações sobre o problema"

#: ../debian/tmp/usr/bin/apport-cli.py:252 ../bin/apport-cli.py:252
msgid ""
"The collected information is being sent to the bug tracking system.\n"
"This might take a few minutes."
msgstr ""
"A informação coletada está sendo enviada ao sistema de rastreamento de "
"erros.\n"
"Isso pode demorar alguns minutos."

#: ../debian/tmp/usr/bin/apport-cli.py:302 ../bin/apport-cli.py:302
msgid "&Done"
msgstr "&Pronto"

#: ../debian/tmp/usr/bin/apport-cli.py:308 ../bin/apport-cli.py:308
msgid "none"
msgstr "nenhum"

#: ../debian/tmp/usr/bin/apport-cli.py:309 ../bin/apport-cli.py:309
#, python-format
msgid "Selected: %s. Multiple choices:"
msgstr "Selecionado: %s. Multiplas opções:"

#: ../debian/tmp/usr/bin/apport-cli.py:325 ../bin/apport-cli.py:325
msgid "Choices:"
msgstr "Escolhas:"

#: ../debian/tmp/usr/bin/apport-cli.py:339 ../bin/apport-cli.py:339
msgid "Path to file (Enter to cancel):"
msgstr "Caminho para o arquivo (Enter para cancelar):"

#: ../debian/tmp/usr/bin/apport-cli.py:345 ../bin/apport-cli.py:345
msgid "File does not exist."
msgstr "O arquivo não existe."

#: ../debian/tmp/usr/bin/apport-cli.py:347 ../bin/apport-cli.py:347
msgid "This is a directory."
msgstr "Isto é um diretório."

#: ../debian/tmp/usr/bin/apport-cli.py:353 ../bin/apport-cli.py:353
msgid "To continue, you must visit the following URL:"
msgstr "Para continuar, você deve visitar a seguinte URL:"

#: ../debian/tmp/usr/bin/apport-cli.py:355 ../bin/apport-cli.py:355
msgid ""
"You can launch a browser now, or copy this URL into a browser on another "
"computer."
msgstr ""
"Você pode executar um navegador agora, ou copiar esta URL dentro de um "
"navegador em outro computador."

#: ../debian/tmp/usr/bin/apport-cli.py:357 ../bin/apport-cli.py:357
msgid "Launch a browser now"
msgstr "Executar o navegador agora"

#: ../debian/tmp/usr/bin/apport-cli.py:371 ../bin/apport-cli.py:371
msgid "No pending crash reports. Try --help for more information."
msgstr ""
"Nenhum relatório de travamentos pendente. Tente --help para maiores "
"informações."

#: ../data/apportcheckresume.py:68
#: ../debian/tmp/usr/share/apport/apportcheckresume.py:68
msgid ""
"This occured during a previous suspend and prevented it from resuming "
"properly."
msgstr ""
"Isto ocorreu durante uma suspensão anterior e preveniu um reinício correto."

#: ../data/apportcheckresume.py:70
#: ../debian/tmp/usr/share/apport/apportcheckresume.py:70
msgid ""
"This occured during a previous hibernate and prevented it from resuming "
"properly."
msgstr ""
"Isto ocorreu durante uma hibernação anterior e preveniu um reinício correto."

#: ../data/apportcheckresume.py:75
#: ../debian/tmp/usr/share/apport/apportcheckresume.py:75
msgid ""
"The resume processing hung very near the end and will have appeared to have "
"completed normally."
msgstr ""
"O processo foi terminado muito perto do final, e nos parece que foi "
"completado normalmente."

#: ../debian/tmp/usr/bin/apport-retrace.py:31 ../bin/apport-retrace.py:31
msgid "%prog [options] <apport problem report | crash ID>"
msgstr "%prog [opções] <relatório de problema apport | id da falha>"

#: ../debian/tmp/usr/bin/apport-retrace.py:33 ../bin/apport-retrace.py:33
msgid "Do not put the new traces into the report, but write them to stdout."
msgstr ""
"Não colocar as novas análises no relatório, mas mostrá-las na saída padrão."

#: ../debian/tmp/usr/bin/apport-retrace.py:35 ../bin/apport-retrace.py:35
msgid ""
"Start an interactive gdb session with the report's core dump (-o ignored; "
"does not rewrite report)"
msgstr ""
"Iniciar uma sessão interativa do gdb com o core dump do relatório (-o "
"ignorado; não reescreve o relatório)"

#: ../debian/tmp/usr/bin/apport-retrace.py:37 ../bin/apport-retrace.py:37
msgid ""
"Write modified report to given file instead of changing the original report"
msgstr ""
"Escrever relatório modificado para determinado arquivo ao invés de alterar o "
"relatório original"

#: ../debian/tmp/usr/bin/apport-retrace.py:39 ../bin/apport-retrace.py:39
msgid "Remove the core dump from the report after stack trace regeneration"
msgstr ""
"Remove o core dump do relatório após a regeneração da análise da pilha"

#: ../debian/tmp/usr/bin/apport-retrace.py:41 ../bin/apport-retrace.py:41
msgid "Override report's CoreFile"
msgstr "Sobrescrever os relatórios \"CoreFile\""

#: ../debian/tmp/usr/bin/apport-retrace.py:43 ../bin/apport-retrace.py:43
msgid "Override report's ExecutablePath"
msgstr "Sobrescrever os relatórios \"ExecutablePath\""

#: ../debian/tmp/usr/bin/apport-retrace.py:45 ../bin/apport-retrace.py:45
msgid "Override report's ProcMaps"
msgstr "Sobrescrever os relatórios \"ProcMaps\""

#: ../debian/tmp/usr/bin/apport-retrace.py:47 ../bin/apport-retrace.py:47
msgid "Rebuild report's Package information"
msgstr "Reconstruir o relatório de informação do pacote"

#: ../debian/tmp/usr/bin/apport-retrace.py:49 ../bin/apport-retrace.py:49
msgid ""
"Build a temporary sandbox and download/install the necessary packages and "
"debug symbols in there; without this option it assumes that the necessary "
"packages and debug symbols are already installed in the system. The argument "
"points to the packaging system configuration base directory; if you specify "
"\"system\", it will use the system configuration files, but will then only "
"be able to retrace crashes that happened on the currently running release."
msgstr ""
"Constrói uma área local temporária e baixa/instala os pacotes necessários e "
"símbolos de depuração lá; sem esta opção ele supõe que que os pacotes "
"necessários e símbolos de depuração já estão instalados no sistema. O "
"argumento aponta para o diretório base de configuração do sistema de "
"empacotamento; se você especificar \"sistema\", ele usará os arquivos de "
"configuração do sistema, mas então será capaz de rastrear somente falhas que "
"ocorreram na versão em execução no momento."

#: ../debian/tmp/usr/bin/apport-retrace.py:51 ../bin/apport-retrace.py:51
msgid ""
"Report download/install progress when installing packages into sandbox"
msgstr ""
"Relatar progresso ao baixar/instalar durante instalação do pacotes na área "
"local"

#: ../debian/tmp/usr/bin/apport-retrace.py:53 ../bin/apport-retrace.py:53
msgid "Prepend timestamps to log messages, for batch operation"
msgstr "Inserir hora antes das mensagens de registro, para operações em lote"

#: ../debian/tmp/usr/bin/apport-retrace.py:55 ../bin/apport-retrace.py:55
msgid "Cache directory for packages downloaded in the sandbox"
msgstr "Diretório de cache para pacotes baixados na área local"

#: ../debian/tmp/usr/bin/apport-retrace.py:57 ../bin/apport-retrace.py:57
msgid ""
"Directory for unpacked packages. Future runs will assume that any already "
"downloaded package is also extracted to this sandbox."
msgstr ""
"Diretório para pacotes descompactados. Execuções futuras assumirão que "
"qualquer pacote obtido via download também será extraído nesta área local."

#: ../bin/apport-valgrind.py:66 ../debian/tmp/usr/bin/apport-retrace.py:59
#: ../bin/apport-retrace.py:59 ../debian/tmp/usr/bin/apport-valgrind.py:66
msgid ""
"Install an extra package into the sandbox (can be specified multiple times)"
msgstr ""
"Instalar um pacote extra na área local (pode ser especificado múltiplas "
"vezes)"

#: ../debian/tmp/usr/bin/apport-retrace.py:61 ../bin/apport-retrace.py:61
msgid ""
"Path to a file with the crash database authentication information. This is "
"used when specifying a crash ID to upload the retraced stack traces (only if "
"neither -g, -o, nor -s are specified)"
msgstr ""
"Caminho para um arquivo com as informações de autenticação do banco de dados "
"de falhas. Isto é usado quando especificado uma identificação (ID) de erro "
"para carregar o rastreamento de pilha refeito (somente se não -g, -o, nem -s "
"forem especificadas)"

#: ../debian/tmp/usr/bin/apport-retrace.py:63 ../bin/apport-retrace.py:63
msgid ""
"Display retraced stack traces and ask for confirmation before sending them "
"to the crash database."
msgstr ""
"Exibir rastreamentos de pilhas refeitos e pedir confirmação antes de enviá-"
"los para o banco de dados de falhas."

#: ../debian/tmp/usr/bin/apport-retrace.py:65 ../bin/apport-retrace.py:65
msgid ""
"Path to the duplicate sqlite database (default: no duplicate checking)"
msgstr ""
"Caminho para o banco de dados de duplicados do SQLite (padrão: nenhuma "
"verificação duplicação)"

#: ../debian/tmp/usr/bin/apport-retrace.py:70 ../bin/apport-retrace.py:70
msgid "incorrect number of arguments; use --help for a short help"
msgstr "número de argumentos incorretos; use --help para uma ajuda rápida"

#: ../debian/tmp/usr/bin/apport-retrace.py:76 ../bin/apport-retrace.py:76
msgid "You cannot use -C without -S. Stopping."
msgstr "Você não pode usar -C sem -S. Interrompendo."

#: ../debian/tmp/usr/bin/apport-retrace.py:80 ../bin/apport-retrace.py:80
msgid ""
"you either need to do a local operation (-s, -g, -o) or supply an "
"authentication file (--auth); see --help for a short help"
msgstr ""
"você precisa fazer uma operação local (-s, -g, -o) ou fornecer um arquivo de "
"autenticação (--auth); veja --help para uma ajuda rápida"

#. translators: don't translate y/n, apport currently only checks for "y"
#: ../debian/tmp/usr/bin/apport-retrace.py:114 ../bin/apport-retrace.py:114
msgid "OK to send these as attachments? [y/n]"
msgstr "OK para enviar estes como anexos? [y / n]"

#: ../gtk/apport-gtk.desktop.in.h:1 ../kde/apport-kde.desktop.in.h:1
#: ../kde/apport-kde-mime.desktop.in.h:1
msgid "Report a problem..."
msgstr "Informar um problema..."

#: ../gtk/apport-gtk.desktop.in.h:2 ../kde/apport-kde.desktop.in.h:2
#: ../kde/apport-kde-mime.desktop.in.h:2
msgid "Report a malfunction to the developers"
msgstr "Relatar um mal-funcionamento aos desenvolvedores"

#: ../data/kernel_oops.py:30 ../debian/tmp/usr/share/apport/kernel_oops.py:30
msgid "Your system might become unstable now and might need to be restarted."
msgstr ""
"Seu sistema poderá se tornar instável agora e poderá ser necessário reiniciá-"
"lo."

#: ../debian/tmp/usr/share/apport/apport-gtk.py:157 ../gtk/apport-gtk.py:157
#, python-format
msgid "Sorry, the application %s has stopped unexpectedly."
msgstr "Desculpe, o aplicativo %s parou de funcionar inesperadamente."

#: ../debian/tmp/usr/share/apport/apport-gtk.py:160 ../gtk/apport-gtk.py:160
#, python-format
msgid "Sorry, %s has closed unexpectedly."
msgstr "Desculpe, %s fechou inesperadamente."

#: ../debian/tmp/usr/share/apport/apport-gtk.py:165 ../kde/apport-kde.py:176
#: ../kde/apport-kde.py:217
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_kde.py:110
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_kde.py:254
#: ../debian/tmp/usr/share/apport/apport-kde.py:176
#: ../debian/tmp/usr/share/apport/apport-kde.py:217 ../gtk/apport-gtk.py:165
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:108
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:300
#, python-format
msgid "Sorry, %s has experienced an internal error."
msgstr "Desculpe, %s apresentou um erro interno."

#: ../debian/tmp/usr/share/apport/apport-gtk.py:186 ../kde/apport-kde.py:170
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_kde.py:292
#: ../debian/tmp/usr/share/apport/apport-kde.py:170 ../gtk/apport-gtk.py:186
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:412
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:437
msgid "Send"
msgstr "Enviar"

#: ../gtk/apport-gtk.ui.h:7 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:7
#: ../debian/tmp/usr/share/apport/apport-gtk.py:204
#: ../debian/tmp/usr/share/apport/apport-gtk.py:573 ../kde/apport-kde.py:271
#: ../debian/tmp/usr/share/apport/apport-kde.py:271 ../gtk/apport-gtk.py:204
#: ../gtk/apport-gtk.py:573
msgid "Show Details"
msgstr "Mostrar detalhes"

#: ../gtk/apport-gtk.ui.h:10 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:10
#: ../debian/tmp/usr/share/apport/apport-gtk.py:216
#: ../debian/tmp/usr/share/apport/apport-gtk.py:284
#: ../debian/tmp/usr/share/apport/apport-gtk.py:303 ../kde/apport-kde.py:210
#: ../kde/apport-kde.py:223
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_kde.py:115
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_kde.py:141
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_kde.py:153
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_kde.py:186
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_kde.py:262
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_kde.py:332
#: ../debian/tmp/usr/share/apport/apport-kde.py:210
#: ../debian/tmp/usr/share/apport/apport-kde.py:223 ../gtk/apport-gtk.py:216
#: ../gtk/apport-gtk.py:284 ../gtk/apport-gtk.py:303
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:115
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:144
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:157
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:192
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:310
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:350
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:479
msgid "Continue"
msgstr "Continuar"

#: ../debian/tmp/usr/share/apport/apport-gtk.py:228 ../gtk/apport-gtk.py:228
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:276
msgid "Force Closed"
msgstr "Forçar fechamento"

#: ../debian/tmp/usr/share/apport/apport-gtk.py:229
#: ../debian/tmp/usr/share/apport/apport-gtk.py:288 ../kde/apport-kde.py:214
#: ../kde/apport-kde.py:361
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_kde.py:226
#: ../debian/tmp/usr/share/apport/apport-kde.py:214
#: ../debian/tmp/usr/share/apport/apport-kde.py:361 ../gtk/apport-gtk.py:229
#: ../gtk/apport-gtk.py:288
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:232
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:273
msgid "Relaunch"
msgstr "Reexecutar"

#: ../debian/tmp/usr/share/apport/apport-gtk.py:238 ../gtk/apport-gtk.py:238
#, python-format
msgid "The application %s has stopped responding."
msgstr "O aplicativo %s parou de responder."

#: ../debian/tmp/usr/share/apport/apport-gtk.py:242 ../gtk/apport-gtk.py:242
#, python-format
msgid "The program \"%s\" has stopped responding."
msgstr "O programa \"%s\" parou de responder."

#: ../debian/tmp/usr/share/apport/apport-gtk.py:257 ../kde/apport-kde.py:184
#: ../debian/tmp/usr/share/apport/apport-kde.py:184 ../gtk/apport-gtk.py:257
#, python-format
msgid "Package: %s"
msgstr "Pacote: %s"

#: ../debian/tmp/usr/share/apport/apport-gtk.py:264 ../kde/apport-kde.py:190
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_kde.py:136
#: ../debian/tmp/usr/share/apport/apport-kde.py:190 ../gtk/apport-gtk.py:264
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:137
msgid "Sorry, a problem occurred while installing software."
msgstr "Desculpe, ocorreu um problema durante a instalação do programa."

#: ../debian/tmp/usr/share/apport/apport-gtk.py:273
#: ../debian/tmp/usr/share/apport/apport-gtk.py:292 ../kde/apport-kde.py:197
#: ../debian/tmp/usr/share/apport/apport-kde.py:197 ../gtk/apport-gtk.py:273
#: ../gtk/apport-gtk.py:292
#, python-format
msgid "The application %s has experienced an internal error."
msgstr "O aplicativo %s apresentou um erro interno."

#: ../debian/tmp/usr/share/apport/apport-gtk.py:276 ../kde/apport-kde.py:201
#: ../debian/tmp/usr/share/apport/apport-kde.py:201 ../gtk/apport-gtk.py:276
#, python-format
msgid "The application %s has closed unexpectedly."
msgstr "O aplicativo %s foi fechado inesperadamente."

#: ../gtk/apport-gtk.ui.h:9 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:9
#: ../debian/tmp/usr/share/apport/apport-gtk.py:287 ../kde/apport-kde.py:213
#: ../debian/tmp/usr/share/apport/apport-kde.py:213 ../gtk/apport-gtk.py:287
msgid "Leave Closed"
msgstr "Deixe fechado"

#: ../gtk/apport-gtk.ui.h:4 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:4
#: ../debian/tmp/usr/share/apport/apport-gtk.py:300 ../kde/apport-kde.py:220
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_kde.py:256
#: ../debian/tmp/usr/share/apport/apport-kde.py:220 ../gtk/apport-gtk.py:300
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:302
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:342
msgid "If you notice further problems, try restarting the computer."
msgstr "Se você observar outros problemas, tente reiniciar o computador."

#: ../debian/tmp/usr/share/apport/apport-gtk.py:304 ../kde/apport-kde.py:224
#: ../debian/tmp/usr/share/apport/apport-kde.py:224 ../gtk/apport-gtk.py:304
msgid "Ignore future problems of this type"
msgstr "Ignorar futuros problemas deste tipo"

#: ../debian/tmp/usr/share/apport/apport-gtk.py:577 ../kde/apport-kde.py:268
#: ../debian/tmp/usr/share/apport/apport-kde.py:268 ../gtk/apport-gtk.py:577
msgid "Hide Details"
msgstr "Ocultar detalhes"

#: ../bin/apport-valgrind.py:37 ../debian/tmp/usr/bin/apport-valgrind.py:37
msgid "See man page for details."
msgstr "Consulte o manual para detalhes."

#: ../bin/apport-valgrind.py:43 ../debian/tmp/usr/bin/apport-valgrind.py:43
msgid "specify the log file name produced by valgrind"
msgstr "especificar o nome do arquivo de log produzido pelo valgrind"

#: ../bin/apport-valgrind.py:46 ../debian/tmp/usr/bin/apport-valgrind.py:46
msgid ""
"reuse a previously created sandbox dir (SDIR) or, if it does not exist, "
"create it"
msgstr ""
"reutilizar um diretório para área local criado anteriormente (SDIR) ou, caso "
"este não exista, criar um"

#: ../bin/apport-valgrind.py:50 ../debian/tmp/usr/bin/apport-valgrind.py:50
msgid ""
"do  not  create  or reuse a sandbox directory for additional debug symbols "
"but rely only on installed debug symbols."
msgstr ""
"não criar ou reutilizar um diretório de área local para símbolos de "
"depuração adicionais, confie apenas nos símbolos de depuração instalados."

#: ../bin/apport-valgrind.py:54 ../debian/tmp/usr/bin/apport-valgrind.py:54
msgid ""
"reuse a previously created cache dir (CDIR) or, if it does not exist, create "
"it"
msgstr ""
"reutilizar um diretório de cache (CDIR) criado previamente ou, caso não "
"exista, criar um"

#: ../bin/apport-valgrind.py:58 ../debian/tmp/usr/bin/apport-valgrind.py:58
msgid ""
"report download/install progress when installing packages into sandbox"
msgstr ""
"reportar download/progresso de instalação ao instalar pacotes na área local"

#: ../bin/apport-valgrind.py:62 ../debian/tmp/usr/bin/apport-valgrind.py:62
msgid ""
"the executable that is run under valgrind's memcheck tool  for memory leak "
"detection"
msgstr ""
"o executável sendo executado pela ferramenta de checagem de memória do "
"valgrind para detecção de vazamento de memória"

#: ../bin/apport-valgrind.py:97 ../debian/tmp/usr/bin/apport-valgrind.py:97
#, python-format
msgid "Error: %s is not an executable. Stopping."
msgstr "Erro: %s não é um executável. Interrompendo."

#: ../kde/apport-kde.py:294 ../debian/tmp/usr/share/apport/apport-kde.py:294
msgid "Username:"
msgstr "Nome de usuário:"

#: ../kde/apport-kde.py:295 ../debian/tmp/usr/share/apport/apport-kde.py:295
msgid "Password:"
msgstr "Senha:"

#: ../kde/apport-kde.py:386 ../debian/tmp/usr/share/apport/apport-kde.py:386
msgid "Collecting Problem Information"
msgstr "Coletando Informações sobre o Problema"

#: ../kde/apport-kde.py:388 ../debian/tmp/usr/share/apport/apport-kde.py:388
msgid ""
"The collected information can be sent to the developers to improve the "
"application. This might take a few minutes."
msgstr ""
"As informações coletadas podem ser enviadas aos desenvolvedores para "
"melhorar aplicativo. Isto pode demorar alguns minutos."

#: ../kde/apport-kde.py:414 ../debian/tmp/usr/share/apport/apport-kde.py:414
msgid "Uploading Problem Information"
msgstr "Enviando Informações sobre o Problema"

#: ../gtk/apport-gtk.ui.h:15 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:15
#: ../kde/apport-kde.py:416 ../debian/tmp/usr/share/apport/apport-kde.py:416
msgid ""
"The collected information is being sent to the bug tracking system. This "
"might take a few minutes."
msgstr ""
"A informação coletada está sendo enviada ao sistema de rastreamento de "
"erros. Isto pode levar alguns minutos."

#: ../gtk/apport-gtk.ui.h:1 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:1
#: ../kde/apport-kde.py:450 ../kde/apport-kde.py:486
#: ../debian/tmp/usr/share/apport/apport-kde.py:450
#: ../debian/tmp/usr/share/apport/apport-kde.py:486
msgid "Apport"
msgstr "Apport"

#: ../apport/com.ubuntu.apport.policy.in.h:1
msgid "Collect system information"
msgstr "Coletar informações sobre o sistema"

#: ../apport/com.ubuntu.apport.policy.in.h:2
msgid ""
"Authentication is required to collect system information for this problem "
"report"
msgstr ""
"É necessário autenticar para coletar informações sobre o sistema para o "
"relatório deste problema"

#: ../apport/com.ubuntu.apport.policy.in.h:3
msgid "System problem reports"
msgstr "Relatórios de problemas do sistema"

#: ../apport/com.ubuntu.apport.policy.in.h:4
msgid ""
"Please enter your password to access problem reports of system programs"
msgstr ""
"Por favor, informe sua senha para acessar os relatórios de problemas dos "
"programas do sistema"

#: ../gtk/apport-gtk.ui.h:2 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:2
msgid "Crash report"
msgstr "Relatório de erros"

#: ../gtk/apport-gtk.ui.h:3 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:3
msgid "<big><b>Sorry, an internal error happened.</b></big>"
msgstr "<big><b>Desculpe, ocorreu um erro interno.</b></big>"

#: ../gtk/apport-gtk.ui.h:5 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:5
msgid "Send an error report to help fix this problem"
msgstr "Envie um relatório de erros para ajudar a corrigir este problema"

#: ../gtk/apport-gtk.ui.h:6 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:6
msgid "Ignore future problems of this program version"
msgstr "Ignorar problemas futuros desta versão do programa"

#: ../gtk/apport-gtk.ui.h:8 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:8
msgid "_Examine locally"
msgstr "_Examinar localmente"

#: ../gtk/apport-gtk.ui.h:11 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:11
msgid "<big><b>Collecting problem information</b></big>"
msgstr "<big><b>Coletando informações sobre o problema</b></big>"

#: ../gtk/apport-gtk.ui.h:12 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:12
msgid ""
"Information is being collected that may help the developers fix the problem "
"you report."
msgstr ""
"A informação que está sendo coletada pode ajudar os desenvolvedores a "
"corrigir o problema que você relatou."

#: ../gtk/apport-gtk.ui.h:14 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:14
msgid "<big><b>Uploading problem information</b></big>"
msgstr "<big><b>Enviando informações sobre o problema</b></big>"

#: ../kde/apport-kde-mimelnk.desktop.in.h:1
msgid "Apport crash file"
msgstr "Arquivo Apport falhou"

#: ../debian/tmp/usr/bin/apport-unpack.py:22 ../bin/apport-unpack.py:22
#, python-format
msgid "Usage: %s <report> <target directory>"
msgstr "Uso: %s <relatório> <diretório de destino>"

#: ../debian/tmp/usr/bin/apport-unpack.py:46 ../bin/apport-unpack.py:46
msgid "Destination directory exists and is not empty."
msgstr "Diretório de destino existe e não está vazio."

#, python-format
#~ msgid "The package \"%s\" failed to install or upgrade."
#~ msgstr "O pacote \"%s\" falhou na instalação ou atualização."

#, no-c-format, python-format
#~ msgid "Complete report (recommended; %s)"
#~ msgstr "Relatório completo (recomendado; %s)"

#~ msgid ""
#~ "<big><b>Send problem report to the developers?</b></big>\n"
#~ "\n"
#~ "After the problem report has been sent, please fill out the form in the "
#~ "automatically opened web browser."
#~ msgstr ""
#~ "<big><b>Enviar relatório de problemas aos desenvolvedores?</b></big>\n"
#~ "\n"
#~ "Após o relatório de problemas ser enviado, preencha o formulário no "
#~ "navegador que se abrirá automaticamente."

#, python-format
#~ msgid "Sorry, %s closed unexpectedly"
#~ msgstr "Desculpe, %s fechou inesperadamente"

#~ msgid "&Details..."
#~ msgstr "&Detalhes..."

#~ msgid "_Send Report"
#~ msgstr "_Enviar Relatório"

#~ msgid "Generic error"
#~ msgstr "Erro genérico"

#~ msgid ""
#~ "After the problem report has been sent, please fill out the form in the "
#~ "automatically opened web browser."
#~ msgstr ""
#~ "Após o relatório de problemas ser enviado, preencha o formulário no "
#~ "navegador que se abrirá automaticamente."

#, python-format
#~ msgid "%s closed unexpectedly on %s at %s."
#~ msgstr "%s fechou inesperadamente em %s em %s."

#~ msgid "Restart _Program"
#~ msgstr "Reiniciar o _Programa"

#, python-format
#~ msgid "Sorry, the package \"%s\" failed to install or upgrade."
#~ msgstr "Desculpe, o pacote \"%s\" falhou na instalação ou atualização."

#, no-c-format
#~ msgid ""
#~ "<span weight=\"bold\" size=\"larger\">Sorry, the package \"%s\" failed to "
#~ "install or upgrade.</span>"
#~ msgstr ""
#~ "<span weight=\"bold\" size=\"larger\">Desculpe, falha ao instalar ou "
#~ "atualizar o pacote \"%s\".</span>"

#, python-format
#~ msgid "&Send complete report (recommended; %s)"
#~ msgstr "&Enviar relatório completo (recomendado; %s)"

#, no-c-format, python-format
#~ msgid "Reduced report (slow Internet connection; %s)"
#~ msgstr "Relatório reduzido (baixa conexão de internet; %s)"

#, python-format
#~ msgid "Sorry, %s closed unexpectedly."
#~ msgstr "Desculpe, %s fechou inesperadamente."

#~ msgid "_Ignore future crashes of this program version"
#~ msgstr "_Ignorar travamentos futuros nesta versão do programa"

#~ msgid "Restart &Program"
#~ msgstr "Reiniciar o &Programa"

#, python-format
#~ msgid "Send &reduced report (slow Internet connection; %s)"
#~ msgstr "Enviar relatório &resumido (conexão lenta com a internet; %s)"

#~ msgid ""
#~ "This will remove some large items from the report. These are very useful for "
#~ "developers to debug the problem, but might be too big for you to upload if "
#~ "you have a slow internet connection."
#~ msgstr ""
#~ "Isto removerá alguns itens grandes do relatório. Estes são muito úteis para "
#~ "que os desenvolvedores solucionem o problema, mas podem ser demasiadamente "
#~ "grandes para que você os envie caso tenha uma conexão lenta com a Internet."

#~ msgid "&Report Problem..."
#~ msgstr "&Reportar Problema..."

#~ msgid "&Send"
#~ msgstr "&Enviar"

#~ msgid "Kernel problem"
#~ msgstr "Problema no kernel"

#~ msgid "&Ignore future crashes of this program version"
#~ msgstr "&Ignorar travamentos futuros desta versão do programa"

#~ msgid ""
#~ "You can help the developers to fix the package by reporting the problem."
#~ msgstr ""
#~ "Você pode ajudar os desenvolvedores a reparar o pacote informando o problema."

#, python-format
#~ msgid "Sorry, the program \"%s\" closed unexpectedly."
#~ msgstr "Desculpe, o programa %s fechou inesperadamente"

#~ msgid "Content of the report"
#~ msgstr "Índice do relatório"

#~ msgid "Package problem"
#~ msgstr "Problema no pacote"

#~ msgid "_Report Problem..."
#~ msgstr "Info_rmar Problema..."

#~ msgid "Your system encountered a serious kernel problem."
#~ msgstr "Seu sistema encontrou um grave problema no kernel"

#~ msgid "You can help the developers to fix the problem by reporting it."
#~ msgstr ""
#~ "Você pode ajudar os desenvolvedores a corrigir o problema, reportando-o."

#~ msgid "Application problem"
#~ msgstr "Problema no aplicativo"

#~ msgid ""
#~ "If you were not doing anything confidential (entering passwords or other "
#~ "private information), you can help to improve the application by reporting "
#~ "the problem."
#~ msgstr ""
#~ "Se você não estava fazendo nada confidencial (inserindo senhas ou a outra "
#~ "informação privada), você pode ajudar a melhorar este aplicativo relatando o "
#~ "problema ocorrido."

#~ msgid ""
#~ "If you were not doing anything confidential (entering passwords or other\n"
#~ "private information), you can help to improve the application by reporting\n"
#~ "the problem."
#~ msgstr ""
#~ "Se você não estava fazendo nada confidencial (digitando senhas ou outras\n"
#~ "informações privadas), você pode ajudar a melhorar o aplicativo relatando\n"
#~ "o problema."

#, python-format
#~ msgid "This is not a genuine %s package"
#~ msgstr "Este não é um pacote %s genuíno"

#~ msgid "Send this data to the developers?"
#~ msgstr "Deseja enviar estes dados para os desenvolvedores?"

#, python-format
#~ msgid "Sorry, the application %s has closed unexpectedly."
#~ msgstr "Desculpe, o aplicativo %s fechou inesperadamente."
